_posts/2018-01-18-From-Manual-to-Automatic.md

```diff
@@ -0,0 +1,111 @@
+---
+layout: post
+authors: Hugo Fernandes, José Ferreira, Sofia Gomes, Elaine Navalho, João Ventura
+title: From Manual to Automatic
+---
+
+In the more recent years the interest on test automation has been increased significantly. Some of the reasons behind this growth include a greater need for faster software delivery processes, increased productivity and cost/effort reduction associated to the tests itself. This article will focus on test automation, its advantages and drawbacks and when and how should the transition from manual to automatic be made. 
+The article will also provide some examples of test automation tools, programming languages used for test automation and continuous integration software.
+
+## How vital is test automation in today’s world. 
+
+We live in a digital age. Every day new applications are deployed in productions systems and product features are improved, adjusted and updated with the objective of promoting the same products in an increasingly competitive market.
+The balance between shorter development cycles, more challenging *time to market* times and quality delivery is nowadays put to the test more than ever. Taking this into mind, many software firms have started to adopt automation methodologies to reduce time and money. In software testing the test automation promotes three significant benefits <sup>[1]</sup>: 
```

Comment:
`more challenging *time to market* times` sounds odd. Couldn't we use use `more challenging *times to market*` instead? According to wikipedia, the plural exists :)

Reply:
Time to market is a metric :), I will try change the statement to remove the "times" word